### ISSUE TITLE
TF Slim Deprecation

### ISSUE THREAD

**Issue by atyshka** [ildoonet/tf-pose-estimation#391]
@ildoonet, are you still actively maintaining this repo? Just wondering since as we advance in tensorflow versions, I believe tf 2.0 coming out this year will remove support for tf slim. Do you have any plans to update this repo so it doesn't break with the new release?

**Comment by ildoonet** [comment]
I will consider to move to tf 2.0. Thanks.

**Comment by atyshka** [comment]
Yeah looking into it it seems everything will have to be moved to Keras. Using frozen graphs is a temporary solution but long term it seems like the Keras changes will be worth it.
